h2o-algos/build.gradle

```diff
@@ -13,8 +13,9 @@ dependencies {
   // Jama dependencies
   compile "gov.nist.math:jama:1.0.3"
   // netlib-java / MTJ dependencies
-  compile 'com.github.fommil.netlib:all:1.1.2'
-  compile('com.googlecode.matrix-toolkits-java:mtj:1.0.4') { exclude group: 'com.github.fommil.netlib' }
+  compile('com.googlecode.matrix-toolkits-java:mtj:1.0.4') {
+    exclude group: 'com.github.fommil.netlib', module: 'all'
```

Comment:
Why is it being removed? It is needed for certain PCA parametrizations I believe. Are you sure it is not needed?

Reply:
right now I'm not sure if it's needed or not(testing). @terrytangyuan introduced this library in December. At this stage I know it compiles fine without it(it is `compile` dependency).

@terrytangyuan Could you please confirm this library is not used?

side note which doesn't have to do anything with the fact if we need it or not -  it's old, not maintained library

Reply:
Oki, so based on the failing tests - we need this lib included. I'll see what is the true cause of the DBC issue in this case

Reply:
Yes this lib is needed for pca. Thanks for checking